_docs/_latest/setup/install.md

```diff
@@ -90,6 +90,12 @@ steps below:
      ```bash
      python install/gcp_installer.py -h
      ```
+     Install Forseti in shared VPC by running the installer with the VPC flags.
+     
+     ```
+     python install/gcp_installer.py --vpc-host-project-id={YOUR_PROJECT_ID} 
```

Comment:
can you change `{YOUR_PROJECT_ID}` to `{VPC_HOST_PROJECT_ID}`

Reply:
Done